samples/ingestion/ingestion-client/Connector/Enums/ConversationPiiCategory.cs

```diff
@@ -0,0 +1,18 @@
+// <copyright file="ConversationPiiCategory.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace Connector.Enums
```

Comment:
imo it should be up to TA to reject requests if there's an unexpected category. 
If you add new entities, you won't be able to pick them up without redeploying the ingestion client

Reply:
I issue I have is we cannot surface these errors when the ARM template is deploying. Is there a way to run that as validations when we are deploying?

Reply:
I think I searched for something like this a while ago and couldn't find anything :(

Reply:
As discussed removed this enum for now.